output/pursuits.json

```diff
@@ -3214,7 +3225,7 @@
   },
   "4126111451": {
     "ActivityMode": [1673724806],
-    "ItemCategory": [153950757]
+    "ItemCategory": [153950757, -153950757]
```

Comment:
value went negative it seems

Reply:
yes b/c this bounty can be completed by both heavy and non-heavy grenade launchers, but some can only be completed by non-heavies which is what the negative denotes.

Reply:
ohhhh ok cool! makes sense.